docker/script/entrypoint.sh

```diff
@@ -69,19 +69,19 @@ fi
 case "$1" in
   local-runner)
     install_requirements
-    airflow initdb
+    airflow db init
```

Comment:
these are all 2.0 code. same comment here. this file should not have any change from v 1.10.12 to .15